v03_pipeline/lib/reference_data/dataset_table_operations.py

```diff
@@ -49,11 +46,7 @@ def get_dataset_ht(
     reference_genome: ReferenceGenome,
 ) -> hl.Table:
     config = CONFIG[dataset][reference_genome.v02_value]
-    ht = (
```

Comment:
I think there's code in `bin/write_cached_reference_dataset_query_ht.py` that could use this function (it'll be dead soon but good to be complete!).